services/uws-api-server/Chart.yaml

```diff
@@ -1,7 +1,10 @@
 apiVersion: v2
 name: uws-api-server
-version: 0.3.0
-dependencies:
-- name: uws-api-server
-  version: 1.3.1
-  repository: https://lsst-dm.github.io/charts/
+version: 1.3.1
```

Comment:
1.4.0 is now current?

Reply:
It is, but it had a bug in it. However, now that the chart is in the ArgoCD repo, it is decoupled from what was in the charts repo. Therefore, I went with the same version so that the ArgoCD app would go be green for sync when the update gets made.